cmd/loadRAGCSV/main.go

```diff
@@ -73,7 +73,7 @@ func batchEmbed(dat [][]string) *genai.BatchEmbedContentsResponse {
 }
 
 func outputEmbeddingsGOB(dat [][]string, res []*genai.BatchEmbedContentsResponse) {
-	o, err := os.Create("../aigogo/internal/vecdb/embeddings.gob")
```

Comment:
Do not move embeddings.go out of the internal package.
If you do then it becomes a public package and we cannot change the structure in the future.